packages/vscode-extension/package.json

```diff
@@ -803,6 +803,12 @@
               {
                 "const": "debug-prepare-manifest",
                 "description": "%teamstoolkit.taskDefinitions.command.manifest.description%"
+              },
+              {
+                "const": "provision"
```

Comment:
Add description?

Reply:
Now these configs are just for test, the lifecycle name may changed.

To avoid release the lifecycle task in the next release, I will removed them before next release and then add these configs and descriptions back.